QueProject/QueProject/Configs/Configurations.swift

```diff
@@ -6,3 +6,12 @@
 //
 
 import Foundation
+
+struct APIConfig {
+    static var baseURL: String {
```

Comment:
[Optimize] write an extension for Bundle.main.object(forInfoDictionaryKey: )
Example: 
static var baseURL = Bundle["BaseURL"]
@BundleProperty(string: "BaseURL") static var baseURL

Refer: https://fxstudio.dev/property-wrapper-trong-10-phut/